### PR TITLE
Correct misspellings

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR corrects misspellings found using the misspellings https://pypi.org/project/misspellings/ tool.

- one RSpec style issue was avoided

_I'm a long-time fan of Prawn, first-time contributor. Thanks for keeping the lights on for all this time!_

### CODE REVIEW

File: spec/prawn/document_grid_spec.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@
       let(:num_columns) { 5 }
       let(:num_rows) { 8 }
       let(:gutter) { 10.0 }
+
```
Comment (pointlessone) [inline spec/prawn/document_grid_spec.rb]: This seem unrelated.

File: spec/prawn/text_spec.rb [hunk 2]
```diff
@@ -66,7 +66,7 @@
       pdf.text ' '
       text = PDF::Inspector::Text.analyze(pdf.render)
       # If anything is rendered to the page, it should be whitespace.
-      text.strings.each { |str| expect(str).to match(/\A\s*\z/) }
+      expect(text.strings).to all(match(/\A\s*\z/))
```
Comment (pointlessone) [inline spec/prawn/text_spec.rb]: I'm not opposing this change but it doesn't seem to be related to misspellings in any way. Maybe at least a separate commit?

**Comment by gettalong** [comment]
@olleolleolle Still interested in getting this merged? If so, could you please rebase onto the latest master commit and address the two issues raised by @pointlessone?

**Comment by pointlessone** [comment]
@olleolleolle Thank you for your contribution.
